x/globalfee/ante/antetest/fee_test_setup.go

```diff
@@ -56,8 +56,8 @@ func (s *IntegrationTestSuite) SetupTestGlobalFeeStoreAndMinGasPrice(minGasPrice
 }
 
 // SetupTestStakingSubspace sets uatom as bond denom for the fee tests.
-func (s *IntegrationTestSuite) SetupTestStakingSubspace(bondDenom string) types.Subspace {
-	s.app.GetSubspace(stakingtypes.ModuleName).Set(s.ctx, stakingtypes.KeyBondDenom, bondDenom)
+func (s *IntegrationTestSuite) SetupTestStakingSubspace(params stakingtypes.Params) types.Subspace {
+	s.app.GetSubspace(stakingtypes.ModuleName).SetParamSet(s.ctx, &params)
 	return s.app.GetSubspace(stakingtypes.ModuleName)
```

Comment:
Hi  @Pantani,  I just realized in sdk, when using `setParamSet`, it will call a validate  function , which will validate if the `bondDenom` is empty, if empty, it will panic. 
this means in our app, when `init`, the staking bondDenom is not empty because `setParamSet` in the call stack of `init`.

so I m not sure now if I still check if bondDenom empty [here](https://github.com/cosmos/gaia/blob/ce9762e3bcbb69d7e08811767da4d522cfd39ae7/x/globalfee/ante/fee.go#L133)
extra checking is always ok ?